### PR TITLE
feat(youtube/theme): explore drawer background

### PR DESCRIPTION
**Pull request by OxrxL**
# TODO:

- [x] Find the constant for light mode

https://www.reddit.com/r/revancedapp/comments/ypt7ii/black_theme_doesnt_get_applied_here_in_the/

### CODE REVIEW

File: app/src/main/java/app/revanced/integrations/patches/LithoThemePatch.java [hunk 1]
```diff
@@ -13,6 +13,7 @@ public class LithoThemePatch {
     };
 
     private static final int[] DARKCONSTANTS = {
+        -14145496, // 'Explore' drawer background
```
Comment (oSumAtrIX) [inline app/src/main/java/app/revanced/integrations/patches/LithoThemePatch.java:16]:
```suggestion
        -14145496, // explore drawer background
```

**Review by oSumAtrIX — COMMENTED** [review]
What about light mode?

**Comment by OxrxL** [comment]
> What about light mode?

This value was found by inotia. I couldn't, because I didn't have this panel (and I still don't have it now).

**Comment by inotia00** [comment]
I also tested it in light mode
there is no litho theme filter to add for light mode

this is the screenshot when i changed the `lightThemeBackgroundColor` to pink

ReVanced2

**Pull request merged**